### DISCUSSION TITLE
Can't see option to schedule publication when using revisions

### DISCUSSION THREAD

**Discussion by MtDalPizzol** [statamic/cms#7291] [Q&A]
Hi, folks! I'm new to Statamic and loving it till now.

I've enabled the revisions feature for a `news` collection and I'm able to publish, unpublish, create revisions and restore revisions.

**HOWEVER**, I can't see the option to **schedule the publication** in the dropdown like shown in [**this screencast**](https://youtu.be/KO8luPtKuyM?t=106).

I've already tried enabling and also disabling `Publish Dates` for the collection but it seems to have no effect.

Is there some configuration I missed to enable publication scheduling?

**Comment by MtDalPizzol** [comment]
Well... I'm not sure if this have changed since that video. But it seems that the "Schedule" option isn't there anymore.

For a post to be scheduled, you need this:

1. Set `Enable Publish Dates` to `true` on the `collection`
2. Ensure you have a `date` field on your entry's blueprint.
3. Set `Future Date Behavior` to `private`
4. Select a future date when editing the entry
5. Save the entry
6. Publish it

This will set `published: true` and `status: scheduled`.

I think this is a bit unintuitive, since when you click Publish... you're not REALY publishing it...
IMO, having the `Schedule` option on the dropdown makes more sense in UI and UX terms.
Took me hours to figure out how entries publication where scheduled since the docs are brief on how date behaviors work and don't mention the interaction between the `published` and `status` properties.

**Comment by jackmcdade** [comment]
Ah unfortunately the Schedule feature got cut before we shipped 3.0 and it hasn't made it its reappearance yet 😅
